packages/calcite-components/src/components/list/list.stories.ts

```diff
@@ -142,18 +174,6 @@ export const scales = (): string => html`
             scale="s"
             slot="actions-start"
           ></calcite-action>
-          <calcite-icon
```

Comment:
@aPreciado88 lets leave these in the demo since it is still valid to use the slots.

Reply:
@driskull I chatted with Aaron about this and the suggestion was to get rid of content slots in favor of iconStart and iconEnd. Should I add all of the content slots back or only add them to a couple of tests?
cc @ashetland

Reply:
We could also use `calcite-avatar` in the content slots. Since we now have icon-start/end it felt best to move icons to the props, leaving the content slots open for custom content.

Reply:
Yeah we can update them to use an avatar if we want. My point is that they are still valid slots and users can use both icon props and content slots.

Reply:
Honestly, we removed a lot of them to try to get that story to load faster, but I believe we were going to log a separate issue to clean up the List stories. This story in particular seems to always show a diff because the screenshot is taken before the page finishes loading.